tools/docker/builder/openwrt/Dockerfile

```diff
@@ -79,39 +91,11 @@ RUN if [ -n "$INTEL_FEED" ] ; then \
     # set them.
 
 ####
-## Prebuilt openwrt and prplmesh dependencies
+## Prebuilt openwrt and prplmesh dependencies, but not the prplMesh ipk itself
 ####
 
 FROM openwrt-prebuilt as prplmesh-builder
 
-# The following args are mandatory, do not expect the build to
-# succeed without specifying them on the command line.
-#
-# TARGET_PLATFORM_TYPE currently needs to be specified, but will be
-# removed in the future.
-ARG TARGET_PLATFORM_TYPE
-# The feed to use to build prplMesh.
-ARG PRPL_FEED
-# The variant to build (nl80211 or dwpal)
-ARG PRPLMESH_VARIANT
-
 USER openwrt
 
-# We first install from the intel feed preferably, because
-# it replaces some packages from the "packages" feed.
-# If no intel feed is present, nothing will be done so it's still ok.
-RUN echo "src-git prpl ${PRPL_FEED}" >> feeds.conf.default \
-    && scripts/feeds update -a \
-    && scripts/feeds install -a -p intel \
-    && scripts/feeds install -a \
-    && echo "CONFIG_PACKAGE_prplmesh${PRPLMESH_VARIANT}=y" >> .config \
-    && make defconfig \
-    && cat .config \
-    && make package/feeds/prpl/prplmesh/compile -j$(nproc) V=sc \
-    && make package/feeds/prpl/prplmesh/compile -j$(nproc) \
-    && make package/feeds/prpl/prplmesh/clean
-    # For some reason, make has to be run twice because
-    # it still rebuilds some packages on the second run.
-    # On the third run, it's able to compile prplmesh only.
-    # V=sc turns on verbosity for the first command to
-    # make debugging easier.
+RUN make package/feeds/prpl/prplmesh/clean
```

Comment:
If that is the only thing that we do here, it doesn't make a lot of sense to make a separate image for it. IOW, this can be merged into openwrt-prebuilt (which should then be renamed to prplmesh-builder, of course).

(nitpick)

Reply:
Indeed, it doesn't make a lot of sense to keep it.
Thanks for the suggestion, done.